contrib/examples/prometheus/README.md

```diff
@@ -0,0 +1,87 @@
+# Prometheus Metrics
+
+Service Catalog maintains metrics through the Prometheus client library and
+exposes them through the Prometheus http adapter at the /metrics endpoint.
```

Comment:
This is the endpoint relative to the controller, right

Reply:
Yes, correct.  I've updated this doc to be explicit.